x-pack/plugins/security_solution/server/lib/detection_engine/signals/search_after_bulk_create.ts

```diff
@@ -166,6 +165,8 @@ export const searchAfterAndBulkCreate = async ({
             }),
           ]);
 
+          // TODO this comment was written when this code lived in a different
+          // TODO location. We should verify whether this is still true
```

Comment:
IMHO, let's just remove these comments if we don't know if they are valid anymore or they don't make any sense. Keeping the TODO's mixed in with the comments is just confusing for anyone else coming into the code vs. just removing the comments or updating them at this point.

Reply:
a99978a